python/lsst/obs/hsc/hscMapper.py

```diff
@@ -84,23 +91,28 @@ def __init__(self, **kwargs):
         #
         self.filters = {}
         for f in [
-            "W-S-G+",
-            "W-S-R+",
-            "W-S-I+",
-            "W-S-Z+",
-            "W-S-ZR",
             "HSC-G",
             "HSC-R",
             "HSC-I",
             "HSC-Z",
             "HSC-Y",
             "ENG-R1",
+            "NB0515",
+            "NB0527",
+            "NB0656",
+            "NB0718",
+            "NB0816",
+            "NB0921",
+            "IB0945",
+            "NB0973",
             "SH",
             "PH",
             "NONE",
             "UNRECOGNISED"]:
             # Get the canonical name -- see #2113
             self.filters[f] = afwImage.Filter(afwImage.Filter(f).getId()).getName()
+        self.defaultFilterName = "UNRECOGNISED"
```

Comment:
Looks like this requires https://github.com/HyperSuprime-Cam/daf_butlerUtils/commit/daee24edba01b01a0412df7f9b4cf70be5b10860 which hasn't been merged to LSST.